src/components/ContentLayout.vue

```diff
@@ -10,13 +10,14 @@
 
 <template>
   <div
-    class="adjustable-sidebar-width"
+    class="content-layout"
     :class="{
       dragging: isDragging,
-      'sidebar-hidden': hiddenOnLarge
+      'sidebar-hidden': !enableSidebar || hiddenOnLarge
     }"
   >
     <div
+      v-if="enableSidebar"
```

Comment:
In case we don't enable the sidebar, we don't need the `BreakpointEmitter` element either. Can we add the conditional `v-if="enableSidebar"` to it, too?